### PR TITLE
Kmee/8.0 nfe refactoring

### PR DESCRIPTION
**Pull request by mileo**

### CODE REVIEW

File: nfe/sped/nfe/processing/processor.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.       #
+###############################################################################
+
+
+from pysped.nfe import ProcessadorNFe as ProcessadorNFePySped
+from pysped.nfe.danfe import DANFE as DanfePySped
+from pysped.nfe.danfe import DAEDE
+
+from .certificado import Certificado
+from memory_profiler import profile
```
Comment (danimaribeiro) [inline nfe/sped/nfe/processing/processor.py]: Just remove this and it's good to go :+1:

**Comment by mileo** [comment]
@danimaribeiro 

Corrigido, podemos fazer o merge?

**Comment by danimaribeiro** [comment]
:+1:

File: nfe/sped/nfe/processing/processor.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        Remove a geração automática do DANFE para deixar o processo + leve
+        :return:
+        """
+        self.conteudo_pdf = ''
+        return
+
+
+class ProcessadorNFe(ProcessadorNFePySped):
+
+    @profile
```
Comment (danimaribeiro) [inline nfe/sped/nfe/processing/processor.py]: Ainda está quebrando o build.

**Comment by mileo** [comment]
@danimaribeiro GREEN!

**Pull request merged**